test/oph/heratepalvelu/amis/AMISherateHandler_test.clj

```diff
@@ -19,17 +19,17 @@
   (reset! call-log (str @call-log "get-koulutustoimija-oid "))
   (:koulutustoimija-oid opiskeluoikeus))
 
-(defn- mock-check-opiskeluoikeus-suoritus-types? [_]
-  (reset! call-log (str @call-log "check-opiskeluoikeus-suoritus-types? "))
+(defn- mock-has-one-or-more-ammatillinen-tutkinto? [_]
+  (reset! call-log (str @call-log "has-one-or-more-ammatillinen-tutkinto? "))
```

Comment:
Näitä on meidän koodipohjassa ihan uskomattomat määrät.
```suggestion
  (swap! call-log #(str % "has-one-or-more-ammatillinen-tutkinto? "))
```

Reply:
tuo deref + reset! oikeasti hukkaa päivityksiä jos näitä tehdään monessa säikeessä yhtaikaa.  Testikoodissa tietysti ihan sama, mut miks nää oikein on kirjoitettu näin?